src/styles/blocks/card-block.scss

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+}
+
+.container {
```

Comment:
The class name `.container` is too generic and might conflict with other styles. Consider using a more specific class name to avoid potential conflicts.